docs/backup-restore-integration.md

```diff
@@ -285,3 +285,146 @@ PersistentVolumeClaims ***owned by DataVolumes*** must have the following ***ann
 ```yaml
 cdi.kubevirt.io/storage.populatedFor: <datavolume name>
 ```
+
+## Validate backup partner compatibility
+In this section, we will describe the different scenarios a backup partner should test in order to assess its compatibility with Kubevirt.
+
+### Prerequisites
+Cluster with the backup software and storage existing and Kubevirt + CDI installed.
+
+To validate there are two options:
+### Automatically
+This requires some extra effort.
+The tests are in our kubevirt-velero-plugin [repo](https://github.com/kubevirt/kubevirt-velero-plugin/tree/main/tests) marked with a label `PartnerComp`.
+You will need to adjust some env variables and the **backup and restore functions** in those tests.
```

Comment:
Interesting idea to leverage the velerteo tests into a test framework of sorts.  Maybe we can have test args/env vars that are paths to backup/restore scripts that the framework executes?

Reply:
yeah sure, I can work on that :)